schema/core/v2.0.0/metadata.yaml

```diff
@@ -532,6 +532,24 @@ classes:
         description: *desc_tomogram_offset
         range: TomogramOffset
         required: true
+      is_portal_standard:
+        description: Whether the tomogram is a portal standard.
+        range: boolean
+        required: false
+        ifabsent: false
+      is_visualization_default:
+        description: Whether the tomogram is the default for visualization.
+        range: boolean
+        required: true
+        ifabsent: true
+      cross_references:
```

Comment:
Just calling this out because both old and new API's are missing field for tomogram cross references and dates. Is this PR intentionally adding functionality?

Reply:
we do want these